wire-ios-sync-engine/Source/UserSession/Search/SearchRequest.swift

```diff
@@ -26,41 +26,37 @@ public struct SearchOptions: OptionSet {
 
     public static let contacts = SearchOptions(rawValue: 1 << 0)
 
-    /// Users found in your address book.
-
-    public static let addressBook = SearchOptions(rawValue: 1 << 1)
-
     /// Users which are a member of the same team as you.
 
-    public static let teamMembers = SearchOptions(rawValue: 1 << 2)
+    public static let teamMembers = SearchOptions(rawValue: 1 << 1)
```

Comment:
is it normal that the option change here?

Reply:
I was thinking the same. My thought was that if it is not persisted anywhere it should be fine. But I didn't check the project that thoroughly

Reply:
I think it was good to check this was not persisted. For now let's keep it like this, I don't have a strong opinion on changing it to enum

Reply:
I think it is good to keep it as it is. An `OptionSet` seems a good solution to it's purpose.